packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -130,7 +171,7 @@ Defines a global `OktaUtil` JavaScript object that contains methods used to comp
 
 By calling the `OktaUtil.getRequestContext()` method, JavaScript code on your sign-in page can inspect the current request and make decisions based on the target application or other details.
 
-If you're using Okta Identity Engine, the following object is returned by invoking `OktaUtil.getRequestContext()`:
+If you're using the Identity Engine, the following object is returned by invoking `OktaUtil.getRequestContext()`:
```

Comment:
I don't know the style here, but you could maybe use "If you're using Identity Engine..." - but I could be 100% wrong on that. Going from line 211 below that just uses "Classic Engine users get..." -> "Identity Engine users get..." -> "Identity Engine" and not "The Identity Engine"

Reply:
Thank you